TBCCore/TBCCurrencyFormatter.h

```diff
@@ -2,14 +2,17 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
 
 @protocol TBCCurrencyFormatter <NSObject>
-- (NSDecimalNumber *)numberFromString:(NSString *)string;
-- (NSString *)stringFromNumber:(NSNumber *)string;
-- (NSString *)stringForObjectValue:(id)object;
+- (nullable NSDecimalNumber *)numberFromString:(NSString *)string;
+- (nullable NSString *)stringFromNumber:(NSNumber *)string;
```

Comment:
Bit 😿 that this inherits nullability.
[CFNumberFormatter.c#L504-L505](https://github.com/apple/swift-corelibs-foundation/blob/a61b058ed53b00621e7acba4c53959e3ae01a254/CoreFoundation/Locale.subproj/CFNumberFormatter.c#L504-L505)